docs/accessibility/core-concepts/comparing-reports.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: 'Comparing Accessibility Reports | Cypress Documentation'
+description: 'Review the main areas to pay attention to when first reviewing an accessibility report for a Cypress run.'
+sidebar_position: 40
+sidebar_label: Compare reports
+---
+
+<ProductHeading product="accessibility" />
+
+# Compare reports
+
+Accessibility reports from different runs can be compared in the [Branch Review](/cloud/features/branch-review#Getting-Started) area of Cypress Cloud. This allows you to instantly see if any new issues have been introduced, and drill in to see full-page HTML and CSS snapshots showing only the new issues.
+
+See the video below for an example of using Branch Review in a Pull Request:
+
+<DocsVideo
+  src="https://www.youtube.com/embed/k3vq57O4P4M"
+  title="Accessibility branch review demo"
+/>
+
+## Use cases
+
+Comparing the results from different runs is useful in multiple scenarios.
+
+**Key use cases:**
+
+- **Pre-merge checks**: Know if any net-new issues are introduce by UI code changes.
+- **Monitoring changes**: Compare nightly monitoring runs and track down the introduction of new problems caused by underlying changes in the application.
+- **Detecting content issues**: Sometimes content editors can introduce accessibility issues unrelated to code changes. Seeing the example issues presented visually, in context, helps you quickly triage whether you are dealing with a recent code change issue, or a content authorship problem.
+- **Reviewing AI-generated code changes**: The increased use of AI to generate and/or review front-end code creates some increased risks of accessibility regressions making it to production. The increase or decrease of accessibility issues when reviewing a pull request helps you understand the impact of the change.
+- **Tracing the introduction of issues**: With dropdowns for each run, it's easy to rapidly compare different A and B runs to find the exact commit that introduced a problem.
```

Comment:
@marktnoonan I would also add the use case for using branch review is to show that you have RESOLVED an issue that your code changes were looking to resolve - and can easier share that overview with your team to more quickly review code changes.

Reply:
updated